src-cvt.md

```diff
@@ -1,7 +1,7 @@
 # FreeBSD Src Committer Transition Guide
 
 This document is designed to walk people through the conversion
-process from Subversion to Git, written from the doc committer's point
+process from Subversion to Git, written from the source committer's point
```

Comment:
should this be src?

Reply:
I went ahead and spelled it out since it is not a repository URL but prose.